pyspecdata/core.py

```diff
@@ -6918,20 +6932,25 @@ def __init__(self,*args,**kwargs):
         return
     @property
     def function_string(self):
-        r'''A property of the fitdata class which stores a string output of the functional form of the desired fit expression provided in func:`functional_form` in LaTeX format'''
+        r'''A property of the fitdata class which stores a string
+        output of the functional form of the desired fit expression
+        provided in func:`functional_form` in LaTeX format'''
         retval = sympy.latex(self.symbolic_expr).replace('$','')
         return r'$f(%s)='%(sympy.latex(self.fit_axis)) + retval + r'$'
     @function_string.setter
     def function_string(self):
         raise ValueError("You cannot set the string directly -- change the functional_form property instead!")
     @property
     def functional_form(self):
-        r'''A property of the fitdata class which is set by the user, takes as input a sympy expression of the desired fit expression'''
+        r'''A property of the fitdata class which is set by the user,
+        takes as input a sympy expression of the desired fit
+        expression'''
         print("Getting symbolic function")
         return self.symbolic_expr
     @functional_form.setter
     def functional_form(self,sym_expr):
-        r''' The functional form, given as a sympy expression, to which you would like to fit the data.'''
+        r''' The functional form, given as a sympy expression, to
+        which you would like to fit the data.'''
         assert issympy(sym_expr), "for now, the functional form must be a sympy expression!"
```

Comment:
I'm not positive, but I think you only need to give a docstring in the getter (the first definition of functional_form, above, which you've already done)

Reply:
Ok I removed this here and added a little bit more in the getter docstring

Reply:
👍